src/MigrationTools.Host/StartupService.cs

```diff
@@ -66,6 +85,7 @@ public void RunExitLogic()
             _logger.LogInformation("The application ran in {Application_Elapsed} and finished at {Application_EndTime}", _mainTimer.Elapsed.ToString("c"), DateTime.Now.ToUniversalTime().ToLocalTime());
             //Log.CloseAndFlush();
             System.Threading.Thread.Sleep(5000);
+            System.Diagnostics.Process.GetCurrentProcess().Kill();
```

Comment:
is this needed?

Reply:
Maybe not, but I was trying to activate a kill. If one needs to kill the service at any time they can get this object and call this method...

Reply:
Maybe there is a better way.

Reply:
Look into the Environment.Exit method instead of the kill. Check how things behaves with that

Reply:
DONE - looks like it works